### PR TITLE
GEARPUMP-5, Add additional authorization check like checking user-organization for cloudfoundry OAuth2 Authenticator.

### PR DESCRIPTION
**Pull request by clockfly**
Jira issue: https://issues.apache.org/jira/browse/GEARPUMP-5

### CODE REVIEW

File: conf/gear.conf [hunk 1]
```diff
@@ -509,12 +509,28 @@ gearpump-ui {
...
+
+        ## Whether to enable additional authorization check.
+        ## If the user fails the check, then Gearpump would log user out.
+        additional-authenticator-enabled = true
+
+        ## Define how to do additional authorization check. The class should implement
+        ## interface CloudFoundryUAAOAuth2Authenticator.AdditionalAuthenticator
+        additional-authenticator = {
+          "class" = "io.gearpump.services.security.oauth2.impl.CloudFoundryUAAOAuth2Authenticator$OrganizationAccessChecker"
+          "organization-url" = "http://<api server of the UAA>/v2/organizations/<organization-guid>"
```
Comment (karol-brejna-i) [inline conf/gear.conf]: I suggest other name: <cloud foundry api endpoint>

File: conf/gear.conf [hunk 2]
```diff
@@ -509,12 +509,28 @@ gearpump-ui {
         icon = "/icons/uaa.png"
 
         ## The hostname of cloudfoudry UAA server prefixed by "http://" or "https://"
+        ##
         ## !!NOTE!! Please relace uaahost with your actual Cloudfounudry UAA server, the
         ## value set here serves as an example.
         uaahost = "http://login.gearpump.gotapaas.eu"
```
Comment (karol-brejna-i) [inline conf/gear.conf]: Let's do it like we did "organization-url". "<cloud foundry login endpoint">

**Comment by clockfly** [comment]
@karol-brejna-i 
Thanks for the comments. They are fixed now.

**Comment by karol-brejna-i** [comment]
Works as expected. +1

**Pull request merged**